src/archivematicaCommon/lib/archivesspace/client.py

```diff
@@ -130,6 +130,17 @@ def resource_type(self, resource_id):
     def get_record(self, record_id):
         return self._get(record_id).json()
 
+    def get_levels_of_description(self):
+        """
+        Returns an array of all levels of description defined in this ArchivesSpace instance.
+        """
+        if not hasattr(self, 'levels_of_description'):
```

Comment:
Why check `hasattr` instead setting `self.levels_of_description = None` in the constructor?  I can't find a source for why this looks odd to me, though.  [SO](http://stackoverflow.com/a/9748715) has some other approaches for handling `hasattr`.